.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # any build folders
 build*/
+!docs/dev/building/
```

Comment:
what is this for?

Reply:
It was to track that directory.

Reply:
ah, that makes sense : ) since "building" is so overloaded in DF and DFHack, though, I'd suggest moving the files from `docs/dev/building/` to `docs/dev/compile/`

Reply:
I hadn't considered someone reading "build" and thinking -> buildings (with dfhack)

Though, even now, I don't think it is much to worry about since it is nested under development. Nowadays [I think this is fair to say] virtually everyone in pc gaming knows of "builds" of the game too through version numbering.

Reply:
+1 for `compile` or maybe `build`.